src/ui/MLFacet/MLFacetValues/MLFacetValues.ts

```diff
@@ -52,6 +52,10 @@ export class MLFacetValues {
     return !!findWhere(this.facetValues, { state: FacetValueState.selected });
   }
 
+  public get hasIdleValues() {
```

Comment:
What is an idle value? Is it different from unselected?

Reply:
No, Idle is unselected. In the future if excluded is added back the terminology will work just fine. Idle = unselected & unexcluded